botocore/hooks.py

```diff
@@ -345,10 +346,47 @@ def __copy__(self):
 
 
 class EventAliaser(BaseEventHooks):
-    def __init__(self, event_emitter, event_aliases=None):
+    def __init__(self, event_emitter, event_aliases=None,
+                 duplicate_events=None, duplicate_event_whitelist=None):
+        """Wrapper class which aliases event names.
+
+        :type event_emitter: BaseEventHooks
+        :param event_emitter: The event emitter to wrap.
+
+        :type event_aliases: dict
+        :param event_aliases: A mapping event subsection to event
+            subsection, where the key is a subsection to be replaced by the
+            value. These can cover multiple event parts, but must be entire
+            event parts. So if you had the event ``foo.bar.baz`` you could
+            replace ``bar`` or ``foo.bar`` but not ``ba``. By default,
+            all services whose service name or endpoint prefix were
+            different at the time of the service id migration are aliased.
+
+        :type duplicate_events: dict
+        :param duplicate_events: A mapping of endpoint prefix to a list of
```

Comment:
I think we should reword the docstring to not be specific toward endpoint prefixes and serviceId's. I think it is fine to mention it as the default but still should be described as a general mapping sort of like what you did for `event_aliases` and `duplicate_events`.